### PR TITLE
Use separate fetched and fetching on activity

### PR DESCRIPTION
**Pull request by msecret**
As discussed in #547, in order for the activity store to be fetched
both events and logs should be fetched (and fetching same thing).

This changes the store to track each separately and provide the same
fetched and fetching API but based on the two separate things tracked.

### CODE REVIEW

File: static_src/stores/activity_store.js [hunk 1]
```diff
@@ -60,21 +60,33 @@ class ActivityStore extends BaseStore {
     super();
     this._data = new Immutable.List();
     this.subscribe(() => this._registerToActions.bind(this));
+    this.eventsFetched = false;
+    this.eventsFetching = false;
+    this.logsFetched = false;
+    this.logsFetching = false;
+  }
+
+  get fetched() {
```
Comment (jeremiak) [inline static_src/stores/activity_store.js]: I really like this! By making the store interface include these values, we can enable any of the stores to calculate them on their own.

File: static_src/stores/activity_store.js [hunk 2]
```diff
@@ -60,21 +60,33 @@ class ActivityStore extends BaseStore {
     super();
     this._data = new Immutable.List();
     this.subscribe(() => this._registerToActions.bind(this));
+    this.eventsFetched = false;
```
Comment (jeremiak) [inline static_src/stores/activity_store.js]: Minor quibble, but it seems that we've been using `_` prefixed variable names for things that only the store should care about. Should these variables follow that convention or are they intended for public use?

**Comment by jeremiak** [comment]
This looks good, just made the point above about private variables. If that's a moot point, let's merge!

File: static_src/stores/activity_store.js (hunk 2 above)
Comment (msecret) [inline static_src/stores/activity_store.js]: Agreed they should be private

Comment (jeremiak) [inline static_src/stores/activity_store.js]: Cool, I just made that change

**Comment by jeremiak** [comment]
I made the changes for prefixing private variables with an `_` so merging now

**Pull request merged**